FirebaseDynamicLinks/Tests/Sample/FDLBuilderTestAppObjC/AppDelegate.m

```diff
@@ -92,7 +92,10 @@ - (void)_showDynamicLinkInfo:(FIRDynamicLink *)dynamicLink {
   [alertVC addAction:[UIAlertAction actionWithTitle:@"Dismiss"
                                               style:UIAlertActionStyleCancel
                                             handler:NULL]];
-  [self.window.rootViewController presentViewController:alertVC animated:YES completion:NULL];
+
+  [[UIApplication sharedApplication].keyWindow.rootViewController presentViewController:alertVC
```

Comment:
Curious why this changed from `self` and if the change was related to the IPV4/6 change.

Reply:
No this is not related with the ipv4/6. While testing the changes, I've seen some issues with the test app.
1. After the fresh install, when the app launches and when we try to show the Alert dialog from the AppDelegate when we receive an FDL, the alert was failing with "First responder error: non-key window attempting reload - allowing due to manual keyboard".

2. Alert was failing when universal linking when the app is started as fresh instead of opening from background.